js/app/articleHTMLRenderer.js

```diff
@@ -179,134 +69,51 @@ var ArticleHTMLRenderer = new Knowledge.Class({
         // "Legacy" content, including most wiki sources and also 2.6 Prensa Libre content,
         // is templated and styled on the client.
 
-        if (model.is_server_templated) {
-            return this._get_html(model);
-        } else {
-            switch (model.source) {
-            case 'wikipedia':
-            case 'wikibooks':
-            case 'wikisource':
-            case 'wikihow':
-                return this._render_legacy_content(model);
-            case 'prensa-libre':
-                return this._render_prensa_libre_content(model);
-            default:
-                return null;
-            }
-        }
+        return this._renderer.render_content(this._get_html(model),
+                                             model.is_server_templated,
```

Comment:
Coding style nitpick: indent only to 4 spaces and keep arguments on the same line where possible.

Reply:
In JS we don't indent to the opening bracket, only 4 spaces. That allows keeping more arguments on the same line without overrunning a sensible character limit.